CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - JWT payload now contains user global id - #5039 by @salwator
 - Made middleware path resolving lazy and refactored middleware names - #5041 by @NyanKiyoshi
 - Generate slug in attribute value save - #5055 by @fowczarek
+- Change bash to sh - #5064 by @drozdowsky
```

Comment:
I don't think this is needed to be put in the changelog

Reply:
Ok, removed